src/kernel/mem/util.c

```diff
@@ -0,0 +1,15 @@
+#include <util.h>
+#include <bitmap.h>
+
+size_t get_number_of_pages_from_size(size_t length) {
```

Comment:
this function is actually just ALIGN_UP(length, pagesize) / pagesize - if you wanted to reuse the next function